Vc/detail/storage.h

```diff
@@ -618,7 +622,343 @@ inline std::basic_ostream<CharT> &operator<<(std::basic_ostream<CharT> & s,
 }
 
 //}}}1
-#endif  // Vc_HAVE_SSE
+#elif defined Vc_HAVE_NEON
```

Comment:
OK, we talked about this. We need to get rid of this copy ASAP. Not for this PR, but that should probably be the next PR.